src/partials/advantages.html

```diff
@@ -1,55 +1,53 @@
-<link href="../css/main.css" rel="stylesheet" />
-<section class="advantages">
-  <div class="advantages__container">
-    <ul class="advantages__list">
-      <li class="advantages__item">
-        <article class="advantages__fact">
-          <img
-            class="advantages__fact-img"
-            src="../images/vanilla-icecream.png"
-            srcset="../images/vanilla-icecream@2x.png"
-            alt="vanilla ice cream icon"
-            width="33"
-            height="50"
-          />
-          <h3 class="advantages__fact-title">721</h3>
-          <p class="advantages__fact-text">
-            Aliquam ac dui vel dui vulputate consectetur. Mauris massa.
-          </p>
-        </article>
-      </li>
-      <li class="advantages__item">
-        <article class="advantages__fact">
-          <img
-            class="advantages__fact-img"
-            src="../images/apple-icecream.png"
-            srcset="../images/apple-icecream@2x.png"
-            alt="vanilla ice cream icon"
-            width="33"
-            height="50"
-          />
-          <h3 class="advantages__fact-title">16kg</h3>
-          <p class="advantages__fact-text">
-            Aliquam ac dui vel dui vulputate consectetur. Mauris massa.
-          </p>
-        </article>
-      </li>
-      <li class="advantages__item">
-        <article class="advantages__fact">
-          <img
-            class="advantages__fact-img"
-            src="../images/heart-icecream.png"
-            srcset="../images/heart-icecream@2x.png"
-            alt="vanilla ice cream icon"
-            width="33"
-            height="50"
-          />
-          <h3 class="advantages__fact-title">84</h3>
-          <p class="advantages__fact-text">
-            Aliquam ac dui vel dui vulputate consectetur. Mauris massa.
-          </p>
-        </article>
-      </li>
-    </ul>
-  </div>
+<link href="../sass/main.css" rel="stylesheet" />
+<section class="advantages advantages__container">
```

Comment:
ten div był wcześniej dobrze. Najpierw robi się sekcje potem wewnątrz div z contenerem. W klasie .advanteges będziesz miała tło które ma się rozszerzać z ekranem. A klasa &__container ma ograniczyć zawartość sekcji, ale nie ruszać tła. W jednym tagu będą się nadpisywać.

Reply:
to wraca div w takim razie.